From e72a92080689453aad38b44f47961387d572cde6 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 05 Jun 2022 15:21:33 +0300
Subject: [PATCH] Revert "[SW-0] common: add wfe for noret call"

Revert rom recovery implementation for secured firmware update via
inband and OOB.

This reverts commit 3e8bf88153fa12c6117a068cba157da29506f3af.

Change-Id: I6b55038a2b6ef0d3680a2079b08086637f764dea
---

diff --git a/common/module_fwupd.c b/common/module_fwupd.c
index 68f1d34..4dc3d18 100644
--- a/common/module_fwupd.c
+++ b/common/module_fwupd.c
@@ -648,5 +648,4 @@
 void module_fwupd_run_noret(void)
 {
 	fit_run_image();
-	wfe();
 }
diff --git a/common/module_i2c_slave.c b/common/module_i2c_slave.c
index 0595c7f..827065e 100644
--- a/common/module_i2c_slave.c
+++ b/common/module_i2c_slave.c
@@ -582,6 +582,4 @@
 		i2cslave_reset_dev();
 	else
 		i2cslave_fit_run_image();
-
-	wfe();
 }
